kover-gradle-plugin/docs/index.md

```diff
@@ -85,223 +799,379 @@ buildscript {
+
+To calculate the coverage value, units are grouped by various entities.
+By default, all application units of code are grouped by a single application entity, so one coverage value is calculated for the entire application using the aggregating function.
+
+But you can group code units by other named entities.
+The `GroupingEntityType` type is used for this:
+- `APPLICATION` - one current coverage value for the entire application will be calculated
+- `CLASS` - the coverage value will be calculated individually for each class. So the bounds will be checked for each class
+- `PACKAGE` - the coverage value will be calculated individually for all classes in each package. So the bounds will be checked for each package
+
+#### Verification rules
+Verification rule - is a set of restrictions on the coverage value for each group.
+Rules can have their own names - they are printed when restrictions are violated (does not work for JaCoCo).
+
+Each restriction represents a bound for valid coverage value, expressed in the minimum and/or maximum allowable values.
+
+Rules can be specified for all report variants:
 ```kotlin
-koverReport {
-    filters {
-        // filters for reports of all build variants
+kover {
+    reports {
+        verify {
+            rule {
+                minBound(50)
+            }
+        }
     }
+}
+```
 
-    verify {
-        // verification rules for all report variants
-    }
+#### Specify verification rules for total variant
+Verification rules for all classes and tests of Gradle build
 
-    androidReports("release") {
-        filters {
-            // override report filters for all reports for `release` build variant
-            // all filters specified by the level above cease to work
+```kotlin
+kover {
+    reports {
+        total {
+            verify {
+                // verification rules
+            }
         }
-        
-        xml { /* XML report config for `release` build variant */ }
-        html { /* HTML report config for `release` build variant */ }
-        verify { /* verification config for `release` build variant */ }
-        log { /* logging config for `release` build variant */ }
     }
 }
 ```
 
-### Kotlin multiplatform project
-These are projects that use Kotlin Multiplatfrom Gradle plugin (Kotlin MPP).
-Working with such projects is identical to working with JVM and Android projects simultaneously, which are described above.
+#### Specify verification rules for named variant
+
+Configuring verification rules for named report variant `custom`
+```kotlin
+kover {
+    reports {
+        variant("custom") {
+            verify {
+                // verification rules
+            }
+        }
+    }
+}
+```
 
-Both JVM and Android targets may be present in such projects, depending on the declared targets, it is necessary to use the appropriate Kover tasks.
+You can read more about creating verification rules in the [corresponding section](#verification).
```

Comment:
Given that this sentence is literally in #verification section, it seems unnecessary